Neo4j.Driver/Neo4j.Driver/Internal/Result/ResultCursorBuilder.cs

```diff
@@ -139,7 +144,20 @@ public async ValueTask<IResultSummary> ConsumeAsync()
     public void RunCompleted(long queryId, string[] fields, IResponsePipelineError error)
     {
         _queryId = queryId;
-        _fields = fields;
+
+        if (fields is not null)
```

Comment:
We probably don't need this check. If we want to include it, we should instead throw a invalid or protocol exception if fields is null

Reply:
There are a couple of places where null is passed in as the `fields` parameter - `OnFailure` and `OnIgnored`. The old code didn't use to do anything with the array so the check wasn't needed, but now it builds the lookups it needs the check.